tools/cell_census_builder/experiment_builder.py

```diff
@@ -435,7 +435,7 @@ def _accumulate_all_X_layers(
         pres_data = raw_X.sum(axis=0) > 0
         if isinstance(pres_data, np.matrix):
             pres_data = pres_data.A
-        pres_data = pres_data[0]
+            pres_data = pres_data[0]
```

Comment:
@bkmartinjr can you double check this? In the test case (probably due to the fact that we're using dense matrices), `pres_data` is a ndarray so this line won't work. I think this should not cause any issues.

Reply:
Given that we are testing the census builder, and it always requires Sparse matrices, the tests should do the same (i.e. the tests should not use dense matrices).

that said, I don't see why this code would break if you use dense matrices, and I suspect something else is going on here.  TL;DR - the change looks incorrect, or at least not addressing the core issue.

Reply:
Which tests fails?

Reply:
The builder itself fails due to (in my case) `pres_data` being an ndarray and not requiring the additional unpacking. I can rewrite the tests using sparse matrices and see if this addresses the issue.

Reply:
Moving to a sparse matrix solved the issue. I'm wondering if it's worth adding that as an explicit constraint?